src/apps/advanced-search/helpers.tsx

```diff
@@ -0,0 +1,110 @@
+import { MultiselectOption } from "../../core/utils/types/multiselect-types";
+import {
+  AdvancedSearchFilterData,
+  AdvancedSearchQuery,
+  AdvancedSearchRowData,
+  advancedSearchFilters
+} from "./types";
+
+const getSpace = (currentText: string) => {
```

Comment:
I really think it would be nice with unit tests for these helpers, although it is probably out of scope?

Reply:
Correct! And correct! 
I need to look into how we can and do do unit testing first.